docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -230,13 +230,23 @@ function registerTentacle() {
     tentacle "${ARGS[@]}"
 }
 
+function run() {
+    echo "==============================================="
+    echo "Starting Octopus Deploy Kubernetes Tentacle"
+    echo "==============================================="
+
+    # the exec here means that the host bash process is replaced by the tentacle process
+    exec tentacle agent --instance Tentacle --noninteractive
+}
+
+
 function verifyTentacleIsNotRegistered() {
   setupVariablesForRegistrationCheck
   getStatusOfRegistration
 
   if [ "$IS_REGISTERED" == "true" ]; then
       echo "Tentacle is already configured and registered with server."
-      exit 0
+      run
```

Comment:
Do you still need this if we have `run` at the bottom?

Reply:
yeah, otherwise we need to rejigger the logic flow so it doesn't run the configuration steps again. After the `exec` line, this script is stopped

Reply:
Ah ok... conceptually it's like this right?

```
isRegistered = HasBeenRegistered()

if (!isRegistered)
{
  configure....
}

run
```

How much effort would it be to rejig the flow?

Reply:
conceptually yes. Hrm, I don't think it would be _that_ hard 🤔
We can do another PR :)